nginx/conf.d/redirects/redirects.conf

```diff
@@ -7041,4 +5116,13 @@ rewrite ^/sign-up/how-to-market-a-rehab-center-reviews/?$ https://$host/sign-up/
 rewrite ^/sign-up/addiction-treatment-marketing-trust-signals/?$ https://$host/sign-up/how-to-market-a-rehab-center/trust-signals/ permanent;
 #DIR-8720 (Automated)
 rewrite ^/new-jersey-hotline/?$ https://$host/rehab-hotline/new-jersey/ permanent;
-rewrite ^/nevada-hotline/?$ https://$host/rehab-hotline/nevada/ permanent;
\ No newline at end of file
+rewrite ^/nevada-hotline/?$ https://$host/rehab-hotline/nevada/ permanent;
+#DIR-6508
+rewrite ^/listings/soba-texas-2528472291/\[https:/goo\.gl/scvqY1\]\(https:/goo\.gl/scvqY1/?$ https://$host/local/? permanent;
+rewrite ^/listings/st\.-peter/?$ https://$host/local/? permanent;
```

Comment:
I don't think you need to add the `?` at the end of the destination url but I guess it doesn't really matter on how it actually redirects

Reply:
I agree but I just re-added them the way they were in prod. There were two batches of redirects that I changed; one with and another without the trailing ?.